plugin/util/mp_helper.py

```diff
@@ -106,6 +105,8 @@ def unload_lib():
 
 
 def native_decoding_supported() -> bool:
+    global _native_lib_handle
+    _native_lib_handle = load_lib()
```

Comment:
I don't want to load the lib here. This function should only report whether the lib was loaded or not.

Reply:
OK. Hopefully this gives you enough info on the underlying cause.